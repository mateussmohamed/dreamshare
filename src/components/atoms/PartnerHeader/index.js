import styled from 'styled-components'

const PartnerHeader = styled.header`
  width: 120px;
  height: 120px;
  position: relative;
`

export default PartnerHeader
