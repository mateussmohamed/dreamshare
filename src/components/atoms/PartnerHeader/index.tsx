import styled from 'styled-components'

export const PartnerHeader = styled.header`
  width: 120px;
  height: 120px;
  position: relative;
`
