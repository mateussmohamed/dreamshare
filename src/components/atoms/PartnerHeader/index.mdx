---
name: PartnerHeader
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import PartnerHeader from './index'

# PartnerHeader
<PropsTable of={PartnerHeader} />

<Playground>
  <PartnerHeader />
</Playground>
