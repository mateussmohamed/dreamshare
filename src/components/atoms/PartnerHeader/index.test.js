import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import PartnerHeader from './index'

describe('[<PartnerHeader />]', () => {
  const wrap = (props = {}) => shallow(<PartnerHeader {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({})

    expect(wrapper).toMatchSnapshot()
  })
})
