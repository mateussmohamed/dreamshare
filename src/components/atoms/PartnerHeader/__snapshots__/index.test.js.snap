// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<PartnerHeader />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "PartnerHeader-sc-40ws0q-0",
        "isStatic": true,
        "rules": Array [
          "width:120px;height:120px;position:relative;",
        ],
      },
      "displayName": "PartnerHeader",
      "render": [Function],
      "styledComponentId": "PartnerHeader-sc-40ws0q-0",
      "target": "header",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
